cl/search/management/commands/cl_update_index.py

```diff
@@ -168,11 +214,23 @@ def add_arguments(self, parser):
             "before starting the processing.",
         )
 
+        parser.add_argument(
+            "--extract_missed_rd",
```

Comment:
```suggestion
            "--extract-unextracted-rds",
```

Reply:
changed by: `extract-and-add-solr-unextracted-rds`